kratos/includes/properties.h

```diff
@@ -254,6 +259,16 @@ class Properties : public IndexedObject
 		mTables[Key(XVariable.Key(), YVariable.Key())] = rThisTable;
     }
 
+    bool const HasTables()
+    {
+        return (mTables.size() > 0);
```

Comment:
You may use `mTables.empty()` as a slightly better implementation. (in some container it is faster)